### PR TITLE
RavenDB-13604 Import TimeSeries doc modified to match generic structure of Bolb

### PR DESCRIPTION
**Pull request by haludi**
https://issues.hibernatingrhinos.com/issue/RavenDB-13604

### CODE REVIEW

File: test/InterversionTests/SmugglerTests.cs [hunk 1]
```diff
@@ -22,512 +11,134 @@ namespace InterversionTests
 {
     public class SmugglerTests : InterversionTestBase
     {
+        //TODO Need to be changed to version with relevant fix
+        const string Server42Version = "4.2.101";
```
Comment (haludi) [inline test/InterversionTests/SmugglerTests.cs]: Maybe worth getting the latest dynamically?

Comment (ayende) [reply test/InterversionTests/SmugglerTests.cs]: Probably best to stay on a fixed version, I think

**Comment by topalach** [comment]
retest this please

**Comment by topalach** [comment]
The previous commit was added to trigger jenkins PR tests.

File: src/Raven.Client/Constants.cs [hunk 2]
```diff
@@ -347,6 +347,13 @@ private Folders()
                     public const string Configuration = "Configuration";
                 }
             }
+
+            public class Blob
```
Comment (ppekrol) [inline src/Raven.Client/Constants.cs]: internal

File: src/Raven.Server/Smuggler/Documents/StreamSource.cs [hunk 3]
```diff
@@ -112,7 +112,7 @@ public DatabaseItemType GetNextType()
                     _log.Operations(msg);
                 _result.AddWarning(msg);
 
-                SkipArray(onSkipped: null, token: CancellationToken.None);
+                SkipArray(onSkipped: null, MaySkipBolb, CancellationToken.None);
```
Comment (ppekrol) [inline src/Raven.Server/Smuggler/Documents/StreamSource.cs]: typo: Blob

File: src/Raven.Server/Smuggler/Documents/StreamSource.cs [hunk 4]
```diff
@@ -707,9 +703,9 @@ public long SkipType(DatabaseItemType type, Action<long> onSkipped, Cancellation
                 case DatabaseItemType.Counters:
 #pragma warning restore 618
                 case DatabaseItemType.CounterGroups:
-                    return SkipArray(onSkipped, token);
+                    return SkipArray(onSkipped, null, token);
                 case DatabaseItemType.TimeSeries:
-                    return SkipTimeSeries(onSkipped, token);
+                    return SkipArray(onSkipped, SkipBolb, token);
```
Comment (ppekrol) [inline src/Raven.Server/Smuggler/Documents/StreamSource.cs]: typo: Blob

File: src/Raven.Server/Smuggler/Documents/StreamSource.cs [hunk 5]
```diff
@@ -888,36 +885,31 @@ private long SkipArray(Action<long> onSkipped, CancellationToken token)
             return count;
         }
 
-        private long SkipTimeSeries(Action<long> onSkipped, CancellationToken token)
+        private void SkipBolb(BlittableJsonReaderObject reader)
```
Comment (ppekrol) [inline src/Raven.Server/Smuggler/Documents/StreamSource.cs]: typo: Blob

File: src/Raven.Server/Smuggler/Documents/StreamSource.cs [hunk 6]
```diff
@@ -888,36 +885,31 @@ private long SkipArray(Action<long> onSkipped, CancellationToken token)
...
-                    count++; //skipping
-                    onSkipped?.Invoke(count);
-
-
-                }
-            }
-
-            return count;
+            if(reader.TryGet(Constants.Documents.Blob.Size, out int size) == false)
+                throw new InvalidOperationException($"Trying to skip BOLB without size specified: doc: {reader}");
```
Comment (ppekrol) [inline src/Raven.Server/Smuggler/Documents/StreamSource.cs]: BLOB

**Review by ppekrol — APPROVED** [review]

**Comment by ppekrol** [comment]
Rebased

**Comment by ppekrol** [comment]
@haludi not all of the tests are passing. I've left (and renamed) the ones that I think are relevant. Can you take a look?

**Pull request merged**